addons/core/translations/form/en-us.yaml

```diff
@@ -149,3 +149,12 @@ ip_addresses:
 sync-interval:
   label: Sync Interval
   help: Number of seconds between the time Boundary syncs hosts in this set.
+choose-path:
+  label: Welcome to Boundary
+  help: You’ve taken the first step towards making your infrastructure more secure. How do you want to get started?
```

Comment:
I realize now this could be moved to the resources translation file. I was thinking of creating it similar to below:
```
onboarding:
  choose-path:
    title: Welcome to Boundary
    description: You’ve taken the first step towards making your infrastructure more secure. How do you want to get started?
  create-resources:
       success: ...
```
Thoughts?